server/storage/backend/backend_test.go

```diff
@@ -122,7 +122,16 @@ func TestBackendBatchIntervalCommit(t *testing.T) {
 }
 
 func TestBackendDefrag(t *testing.T) {
-	b, _ := betesting.NewDefaultTmpBackend(t)
+	bcfg := backend.DefaultBackendConfig()
+	// The goal is to verify that we restore config option after defrag.
+	// Just in case check that we aren't setting default value.
+	if bcfg.BackendFreelistType == bolt.FreelistMapType {
+		t.Error("don't use default BackendFreelistType value")
```

Comment:
It's not good to make any assumption on the default value. You can just set a different value in the test and verify it after the defragmentation operation.